_docs/4-polarfire-soc.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: splash
+title: "PolarFire SoC"
+permalink: /docs/polarfire-soc/
+hidden: true
+header:
+  overlay_color: "#5e616c"
+  overlay_image: /assets/images/201106-CORP-BNR-Design-Centers-FPGAs-and-plds-Banner-2880x280.jpg
+excerpt: "Welcome to the PolarFire SoC Page."
+feature_row:
+  - image_path: /assets/images/terminals-linux-300x500.png
+    alt: "customizable"
+    title: "Linux Developer"
+    excerpt: "Infomation to aid a Linux Developer to program on the Icicle Kit using programs such as Yocto and Buildroot."
```

Comment:
Explore Linux on PolarFire SoC hardware platforms with Yocto, Buildroot and more.

Reply:
Sounds good, I'll make this change now.